pkg/model/fileshare.go

```diff
@@ -84,7 +84,7 @@ type FileShareSpec struct {
 	SnapshotId string `json:"snapshotId,omitempty"`
 
 	// ExportLocations of the fileshare.
-	ExportLocations []string `json:"exportLocations,omitempty"`
+	ExportLocations string `json:"exportLocations,omitempty"`
```

Comment:
I don't understand why this is changed to `string`

Reply:
Currently we are testing on only one network, so made it string. I will remove string and make array. Thanks